chain/stmgr/execute.go

```diff
@@ -99,7 +99,6 @@ func (sm *StateManager) ApplyBlocks(ctx context.Context, parentEpoch abi.ChainEp
 		}
```

Comment:
nit: not at all your bug but reading the above code it looks like line 82's error message is wrong and confusing and makes sense to fix now.

Reply:
I'd rather not touch that for now. I don't think that error is consensus critical, but I'd rather not touch it.